app/src/main/java/com/jerboa/feat/BlurNsfwTypes.kt

```diff
@@ -0,0 +1,27 @@
+package com.jerboa.feat
+
+import android.os.Parcelable
+import androidx.annotation.StringRes
+import com.jerboa.R
+import com.jerboa.datatypes.types.PostView
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+enum class BlurNsfwTypes(@StringRes val resId: Int) : Parcelable {
```

Comment:
Rename this to `BlurNsfw` ... but again this is going to conflict with your real `blur_nsfw` user setting saved in the back end.